AsyncDisplayKit/ASMapNode.mm

```diff
@@ -17,70 +17,80 @@ @interface ASMapNode()
   MKMapSnapshotter *_snapshotter;
   MKMapSnapshotOptions *_options;
   NSArray *_annotations;
-  ASDisplayNode *_mapNode;
   CLLocationCoordinate2D _centerCoordinateOfMap;
 }
 @end
 
 @implementation ASMapNode
 
-@synthesize liveMap = _liveMap;
 @synthesize needsMapReloadOnBoundsChange = _needsMapReloadOnBoundsChange;
 @synthesize mapDelegate = _mapDelegate;
+@synthesize region = _region;
 
-- (instancetype)initWithCoordinate:(CLLocationCoordinate2D)coordinate
+#pragma mark - Lifecycle
+- (instancetype)init
 {
   if (!(self = [super init])) {
```

Comment:
Thanks for noticing / using the unusual init style of ASDK.  Some newer components don't use it, and it's not enforced, but I still prefer this one and use it for my code ;).